packages/neo-one-website/src/components/reference/components/ParameterPropertyList.tsx

```diff
@@ -17,12 +17,12 @@ const ParameterLayout = styled(Box)`
 `;
 
 interface Props {
-  readonly values: ReadonlyArray<Parameter | Property>;
+  readonly values?: ReadonlyArray<Parameter | Property>;
```

Comment:
Why is this optional if you're assigning a default value to it anyway?

Reply:
Its optional just to match the usage of the input which is optional (i'd have to track it down but if you peek references its called like `functiondata.<something>`).